src/main/connector/hardware/MIB2/MIB2.yaml

```diff
@@ -48,9 +48,12 @@ monitors:
         source: ${source::monitors.network.discovery.sources.source(3)}
         attributes:
           id: $1
+          hw.network.id: $1
           __display_id: $8
           physical_address: $4
           physical_address_type: MAC
           device_type: $3
           hw.parent.type: enclosure
-          name:  ${awk::sprintf("%s (%s)", $7, $3)}

```

Comment:
Not sure we mentioned removing the name in the issue description, this leads to display issues in the Hardware Dashboards.